app/services/replication/failure_remediator.rb

```diff
@@ -4,13 +4,14 @@ module Replication
   # Remediates catalog entries, e.g., prunes failed replication records
   # Currently this is only called from rake.
   class FailureRemediator
-    def self.prune_replication_failures(druid:, version:)
-      new(druid: druid, version: version).prune_replication_failures
+    def self.prune_replication_failures(druid:, version:, verify_expiration:)
```

Comment:
Would you consider making this default to `true`? (Current behavior).

Reply:
defaulting to current behavior would be megaspiffy!

Reply:
Sure, is this called from anywhere else but the rake task? It is currently defaulted there, so will always get sent true from the task. I ran into an issue or two testing the default here, but can work through it. Thoughts @justinlittman ?

Reply:
I suspect that we may want to use this elsewhere in the future (e.g., for remediation via a web interface) so I would have a slight preference for providing a default.